workers/gitlab_issues_worker/gitlab_issues_worker.py

```diff
@@ -49,28 +30,10 @@ def __init__(self, config={}):
         self.tool_source = 'Gitlab API Worker'
         self.tool_version = '0.0.0'
         self.data_source = 'GitLab API'
+        self.platform_id = 25150
```

Comment:
Why is this hardcoded?

Reply:
There is not currently a standard way of specifying that variable, and the gitlab_issue worker needs it. Can @ccarterlandis or @gabe-heim  fix this?